src/tactic/linarith/parsing.lean

```diff
@@ -36,6 +36,8 @@ namespace linarith
 /-- Variables (represented by natural numbers) map to their power. -/
 @[reducible] meta def monom : Type := rb_map ℕ ℕ
 
+meta def monom.one : monom := rb_map.mk _ _
```

Comment:
I believe these all need docstrings.  Is there a reason you didn't make `has_one` instances?

Reply:
I didn't make instances because these are only used very locally, it just didn't seem necessary.